packages/appstore/src/components/appstore-banner/appstore-banner.scss

```diff
@@ -0,0 +1,88 @@
+.appstore-banner {
+    &__container {
+        position: relative;
+        margin-bottom: 2.4rem;
+        overflow: hidden;
+        background-color: var(--general-main-1);
+
+        border: 1px solid var(--appstore-banner-border-color);
+        border-radius: $BORDER_RADIUS * 4;
+
+        @include desktop {
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+        }
+
+        @include mobile {
+            padding: 1.4rem 1.6rem;
+            border-radius: $BORDER_RADIUS * 2;
+        }
+    }
+
+    &__image {
+        position: absolute;
+        z-index: 0;
+        right: 0;
+        bottom: 0;
+        border-top-right-radius: $BORDER_RADIUS * 4;
+        border-bottom-right-radius: $BORDER_RADIUS * 4;
+
+        @include mobile {
+            border-top-right-radius: $BORDER_RADIUS * 2;
+            border-bottom-right-radius: $BORDER_RADIUS * 2;
+        }
+    }
+
+    &__svg-migrate-banner {
+        height: 18rem;
+
+        @include mobile {
+            height: 10.8rem;
+        }
+
+        &-description {
+            position: relative;
+            z-index: 2;
+            width: 100%;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: baseline;
+
+            @include desktop {
+                padding-left: 4.8rem;
+            }
```

Comment:
do we need `@include desktop` as it's by default for desktop. Right?

Reply:
padding left is only for desktop view

Reply:
this change is made